adafruit_is31fl3731/__init__.py

```diff
@@ -85,44 +87,35 @@ class IS31FL3731:
     The IS31FL3731 is an abstract class contain the main function related to this chip.
     Each board needs to define width, height and pixel_addr.
 
-    :param ~adafruit_bus_device.i2c_device i2c_device: the connected i2c bus i2c_device
-    :param address: the device address; defaults to 0x74
+    :param ~busio.I2C i2c: the connected i2c bus i2c_device
+    :param int address: the device address; defaults to 0x74
     """
 
     width = 16
     height = 9
 
     def __init__(self, i2c, address=0x74):
-        self.i2c = i2c
-        self.address = address
+        self.i2c_device = I2CDevice(i2c, address)
         self._frame = None
         self._init()
 
     def _i2c_read_reg(self, reg, result):
```

Comment:
This function got changed by another recent PR and the changes will get need to get conflicts resolved with this one before it can get merged.